lib/build.gradle

```diff
@@ -480,6 +481,7 @@ task unitTest(dependsOn: ['smokeTest', 'test'])
 task fullTest(dependsOn: ['unitTest'])
 
 task ciCheck(dependsOn: ['checkstyle', 'pmd', 'findbugsXml', 'test'])
+task ciCheckWindows(dependsOn: ['pmd', 'findbugsXml', 'test'])
```

Comment:
Why no checkstyle on windows?

Reply:
Because the tarball was built based on linux so the line feed format doesn't work with the check style on windows.

Reply:
I will create the issue for this. First I thought about disabling the check but I don't think it's good idea to do that.

Another option is to do the below but I understand that if I check out the code on Windows, that could create the same problem.
 <module name="NewlineAtEndOfFile">
   <property name="lineSeparator" value="lf"/>
 </module>

If you can point me to the right config, I can make change right away.